web-api/migration-terraform/main/lambdas/migration-segments.js

```diff
@@ -165,6 +153,7 @@ exports.handler = async event => {
     ...(await hasMigrationRan('bug-0036-public-served-parties-code.js')),
     ...(await hasMigrationRan('0036-phone-number-format.js')),
```

Comment:
Those check that the migrations have been run in the past, I don't think there's any harm in keeping them there.

Reply:
If we started up a new environment today, it wouldn’t have these migrations ran — and since the migration code has no capability to run these migrations, I don’t believe the code would be deployable. 

Am I missing something? I might be. 🤔

Reply:
Updated!

Reply:
Just circling back here to summarize for the future — we discussed this after standup this morning with @mmarcotte and @ericsorenson. 

Meta-level decisions:

- If migrations are removed, they should be deleted from the codebase with its tests, as well as these lines which call `hasMigrationRan`. Removing the `hasMigrationRan` calls at the same time as the migrations keeps the application deployable.
- Since the Court’s environments receive migrations after Flexion’s, @mmarcotte is responsible for removing outdated migrations going forward, and he doesn’t mind being nudged to do so if they’re getting in the way. 
- Removing migration code reduces clutter in this folder and keeps tests running fast. 

For this pull request:

- This PR should be updated to revert removing the other migrations, leaving them unchanged in the PR. 
- The PR should continue to include the introduction of the new migration needed.